src/applications/disability-benefits/526EZ/Form526EZApp.jsx

```diff
@@ -5,11 +5,12 @@ import formConfig from './config/form';
 
 import ITFWrapper from '../all-claims/containers/ITFWrapper';
 import DisabilityGate from './containers/DisabilityGate';
-import EVSSClaimsGate from '../all-claims/containers/EVSSClaimsGate';
+import BetaGate from './containers/BetaGate';
+// import EVSSClaimsGate from '../all-claims/containers/EVSSClaimsGate';
```

Comment:
Is the plan to keep the `EVSSClaimsGate` around so we can use it again when we take the app out of beta?

Reply:
Yep